contrib/runners/action_chain_runner/action_chain_runner.py

```diff
@@ -193,7 +193,6 @@ def _get_rendered_vars(vars, action_parameters):
         if not vars:
             return {}
         context = {}
-        context.update({SYSTEM_SCOPE: KeyValueLookup(scope=SYSTEM_SCOPE)})
```

Comment:
I assume this change was intentional aka old notation is not supported anymore here (and not just deprecated)?